libcodechecker/libclient/client.py

```diff
@@ -64,9 +62,6 @@ def handle_auth(host, port, username, login=False):
         if auth_token and handshake.sessionStillActive:
             LOG.info("You are already logged in.")
             return
-        else:
-            LOG.info("Server requires authentication to access. Please use "
```

Comment:
Yes. When the user tries to log in, they get this on the `cmd`:

```
$ CodeChecker cmd runs
[ERROR] [18:59] - Access denied. This server requires authentication.
[ERROR] [18:59] - Please log in onto the server using 'CodeChecker cmd login'.
```

This message was removed so that typing in `CodeChecker cmd login` does not prompt like this:

```
$ CodeChecker cmd login
[19:00] - Server requires authentication to access. Please use 'CodeChecker cmd login' to authenticate.
[19:00] - Logging in using credentials from command line...
Please provide password for user 'whisperity'
```

(The first line is removed. It is superfluous to tell the user to use [command] to authenticate right when they are *actually* using [command].)